agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -394,6 +394,45 @@ bool slave_thread::handle_cmdu_control_ieee1905_1_message(Socket *sd,
     return true;
 }
 
+bool slave_thread::start_monitoring(const beerocks_message::sClientMonitoringParams &params,
+                                    std::shared_ptr<beerocks_header> beerocks_header)
+{
+    LOG(DEBUG) << __func__;
+
+    std::string client_mac              = network_utils::mac_to_string(params.mac);
+    std::string client_bridge_4addr_mac = network_utils::mac_to_string(params.bridge_4addr_mac);
+    std::string client_ip               = network_utils::ipv4_to_string(params.ipv4);
+
+    LOG(DEBUG) << "START_MONITORING_REQUEST: mac=" << client_mac << " ip=" << client_ip
+               << " bridge_4addr_mac=" << client_bridge_4addr_mac
+               << " channel=" << std::to_string(params.channel)
```

Comment:
If you're printing channel, better also print the bandwidth. As a matter of fact, now it makes more sense to also print the operating class if its available in this context.